pkg/api/constants/constants.go

```diff
@@ -0,0 +1,10 @@
+package constants
```

Comment:
not sure we want a package for constants... annotation keys typically go in the API package directly